spec/support/request/stripe_stubs.rb

```diff
@@ -37,7 +37,6 @@ def stub_payment_method_attach_request
 
   def stub_retrieve_payment_method_request(payment_method_id = "pm_1234")
     stub_request(:get, "https://api.stripe.com/v1/payment_methods/#{payment_method_id}")
-      .with(headers: { 'Authorization' => 'Bearer sk_test_12345' })
```

Comment:
It's helpful to mention why you changed something instead what you changed. I can see the change in the code but may need some context why it's done or what the goal is.

Reply:
This is great feedback Maikel. Thanks. I always add the "what" in the commit, the why is often, like in this case, very important!
To be honest i dont exactly understand why in some cases the authorization header is not necessary in the mock... but it's probably related to the api_key being set or not in the test case.